src/components/Navigation.tsx

```diff
@@ -1,109 +1,153 @@
 import React from 'react';
 
-import { Flex, Link } from '@chakra-ui/react';
+import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Drawer,
+  DrawerBody,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  Flex,
+  Link,
+  useDisclosure,
+} from '@chakra-ui/react';
 import { Link as GatsbyLink } from 'gatsby';
+import Img from 'gatsby-image';
 
-function Navigation(): JSX.Element {
+interface NavLinkProps {
+  link: string;
+  text: string;
+  onClick?: () => void;
+}
+
+interface NavigationProps {
+  data: GatsbyTypes.LayoutQuery;
+}
+
+function NavLink({ link, text }: NavLinkProps): JSX.Element {
   return (
-    <Flex
-      h="55px"
-      wrap="nowrap"
-      mx="auto"
-      flexDir="row"
-      w="full"
-      justifyContent="space-evenly"
-      alignItems="center"
+    <Link
+      _hover={{
+        textDecoration: 'underline',
+        textDecorationColor: '#65BF73',
+        textDecorationThickness: '3px',
+      }}
+      fontFamily="Avenir"
+      fontWeight="850"
+      fontSize="16px"
+      as={GatsbyLink}
+      to={link}
     >
-      <Link
-        _hover={{
-          textDecoration: 'underline',
-          textDecorationColor: '#65BF73',
-          textDecorationThickness: '3px',
-        }}
-        fontFamily="Avenir"
-        fontWeight="850"
-        fontSize="16px"
-        as={GatsbyLink}
-        to="/"
-      >
-        HOME
-      </Link>
+      {text}
+    </Link>
+  );
+}
 
-      <Link
-        _hover={{
-          textDecoration: 'underline',
-          textDecorationColor: '#65BF73',
-          textDecorationThickness: '3px',
-        }}
-        fontFamily="Avenir"
-        fontWeight="850"
-        fontSize="16px"
-        as={GatsbyLink}
-        to="/about"
-      >
-        ABOUT
-      </Link>
+function MobileNavLink({ link, text, onClick }: NavLinkProps): JSX.Element {
+  return (
+    <Link
+      _hover={{
+        textDecoration: 'none',
+      }}
+      fontFamily="Avenir"
+      fontWeight="850"
+      fontSize="16px"
+      as={GatsbyLink}
+      to={link}
+      borderBottom="1px"
+      borderBottomColor="white"
+      onClick={onClick}
+      py={3}
+    >
+      {text}
+    </Link>
+  );
+}
 
-      <Link
-        _hover={{
-          textDecoration: 'underline',
-          textDecorationColor: '#65BF73',
-          textDecorationThickness: '3px',
-        }}
-        fontFamily="Avenir"
-        fontWeight="850"
-        fontSize="16px"
-        as={GatsbyLink}
-        to="/recipes/"
-      >
-        RECIPES
-      </Link>
+function Navigation({ data }: NavigationProps): JSX.Element {
+  const navigationItems: { [link: string]: string } = {
+    '/': 'HOME',
```

Comment:
These can just be regular case - we can add `text-transform: uppercase` on the actual component to stylize (https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)